<?xml version="1.0"?>
<robot
    xmlns:xacro="http://www.ros.org/wiki/xacro" name="hydra3_common" >

  <xacro:property name="pi" value="3.1415926535897931" />
  <xacro:property name="link_length" value=".44" />
  <xacro:property name="protector_radius" value=".155" />


  <xacro:macro name="link_color">
    <material name="color">
      <color rgba="0.75 .5 .2 1"/>
    </material>
  </xacro:macro>

</robot>
